### ISSUE TITLE
Plugin pulls voice actors for all the languages.

### ISSUE THREAD

**Issue by Roshri** [jellyfin/jellyfin-plugin-anilist#32]
I'd like to find a way to limit which voice actors get pulled to the metadata of a title but I can't find a way to do so, currently when a show matches to an Anilist entry, it pulls every voice actor for a character like this

![image](https://user-images.githubusercontent.com/29184137/173252722-d1d63890-5d80-41b3-9240-535228160872.png)

Ideally, it would only pull the very first one, which is the Japanese one. Is there a way to do this?

**Comment by chu-shen** [comment]
anilist Configuration->check "Filter people by title preference"

**Comment by As4shi** [comment]
> anilist Configuration->check "Filter people by title preference"

This option is kinda useless for the most part, unless you can read kanji or don't mind the titles in kanji. It would be better to have an option to choose if we want the original cast only, only the dubbed version cast or both.

I made some small changes to allow that, they are available on [my fork](https://github.com/As4shi/jellyfin-plugin-anilist) if anyone wants to see/use it. I haven't created a pull request because honestly I didn't have the time to read through the documentation, but if anyone is willing to implement it properly feel free to do so.

**Comment by sannidhyaroy** [comment]
> This option is kinda useless for the most part, unless you can read kanji or don't mind the titles in kanji. It would be better to have an option to choose if we want the original cast only, only the dubbed version cast or both.

I agree. I watch subbed animes and would like the only the Japanese Voice Actors to show up but then I have to set the title to Japanese to get that to happen. It makes the plugin pointless.
